tests/test_clinvar_to_evidence_strings.py

```diff
@@ -54,8 +54,9 @@ def test_consequence_type_dict(self):
 class CreateTraitTest(unittest.TestCase):
```

Comment:
Test cases for multiple mappings for a single trait should be added (I'm not sure if they should be in this class or `GetMappingsTest` though).